.github/actions/frontend/runtime/e2e_run/action.yml

```diff
@@ -20,21 +20,26 @@ inputs:
   commit_info_sha:    
     description: 'The commit sha associated with the repo that triggered the e2e_run'
     type: string
+    required: true
```

Comment:
Will this break all the current E2E users?

Reply:
No, the only consumers of this are our workflows. The two in question now pass this in as an input.